website/server/models/task.js

```diff
@@ -243,14 +243,19 @@ export let HabitSchema = new Schema(_.defaults({
 export let habit = Task.discriminator('habit', HabitSchema);
 
 export let DailySchema = new Schema(_.defaults({
-  frequency: {type: String, default: 'weekly', enum: ['daily', 'weekly']},
+  frequency: {type: String, default: 'weekly', enum: ['daily', 'weekly', 'sinceCompletion', 'timesInInterval']}, // 'Every X Interval', 'Certain Days of the Week', 'Every X Interval Since Completion', 'Y Times in X Interval'
```

Comment:
This field is getting unwieldy. Let's expand it out:

```js
frequency: {
  type: String,
  default: 'weekly',
  enum: [
    'daily', // Every X Interval
    'weekly', // Certain Days of the Week
    'sinceCompletion', // Every X Interval Since Completion
    'timesInInterval', // Y Times in X Interval
  ]
}
```